config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -657,8 +657,11 @@ def generate_misc():
                    k8s_version="ci",
                    networking="calico",
                    feature_flags=["AWSIPv6"],
-                   runs_per_day=12,
+                   runs_per_day=24,
                    extra_flags=['--ipv6',
+                                '--override=cluster.spec.api.loadBalancer.type=Public',
+                                '--override=cluster.spec.api.loadBalancer.class=Network',
+                                '--override=cluster.spec.api.loadBalancer.useForInternalApi=true',
```

Comment:
What's the reason for using this for the internal API? Because dns-controller isn't registering AAAA records?

Reply:
Yes

Reply:
This is In preparation for fixing:
```
ServiceAccounts ServiceAccountIssuerDiscovery should support OIDC discovery of service account issuer [Conformance]
```